modules/packages/HDF5.chpl

```diff
@@ -0,0 +1,3795 @@
+
+    extern proc H5Pget_est_link_info(plist_id : hid_t, ref est_num_entries : c_uint, ref est_name_len : c_uint) : herr_t;
+
+    extern proc H5Pset_link_creation_order(plist_id : hid_t, crt_order_flags : c_uint) : herr_t;
+
+    extern proc H5Pget_link_creation_order(plist_id : hid_t, ref crt_order_flags : c_uint) : herr_t;
+
+    extern proc H5Pset_char_encoding(plist_id : hid_t, encoding : H5T_cset_t) : herr_t;
+
+    extern proc H5Pget_char_encoding(plist_id : hid_t, ref encoding : H5T_cset_t) : herr_t;
+
+    extern proc H5Pset_nlinks(plist_id : hid_t, nlinks : size_t) : herr_t;
+
+    extern proc H5Pget_nlinks(plist_id : hid_t, ref nlinks : size_t) : herr_t;
+
+    extern proc H5Pset_elink_prefix(plist_id : hid_t, prefix : c_string) : herr_t;
+
+    extern proc H5Pget_elink_prefix(plist_id : hid_t, prefix : c_string, size : size_t) : ssize_t;
+
+    extern proc H5Pget_elink_fapl(lapl_id : hid_t) : hid_t;
+
+    extern proc H5Pset_elink_fapl(lapl_id : hid_t, fapl_id : hid_t) : herr_t;
+
+    extern proc H5Pset_elink_acc_flags(lapl_id : hid_t, flags : c_uint) : herr_t;
+
+    extern proc H5Pget_elink_acc_flags(lapl_id : hid_t, ref flags : c_uint) : herr_t;
+
+    extern proc H5Pset_elink_cb(lapl_id : hid_t, func : H5L_elink_traverse_t, op_data : c_void_ptr) : herr_t;
+
+    extern proc H5Pget_elink_cb(lapl_id : hid_t, ref func : H5L_elink_traverse_t, ref op_data : c_void_ptr) : herr_t;
+
+    extern proc H5Pset_copy_object(plist_id : hid_t, crt_intmd : c_uint) : herr_t;
+
+    extern proc H5Pget_copy_object(plist_id : hid_t, ref crt_intmd : c_uint) : herr_t;
+
+    extern proc H5Padd_merge_committed_dtype_path(plist_id : hid_t, path : c_string) : herr_t;
+
+    extern proc H5Pfree_merge_committed_dtype_paths(plist_id : hid_t) : herr_t;
+
+    extern proc H5Pset_mcdt_search_cb(plist_id : hid_t, func : H5O_mcdt_search_cb_t, op_data : c_void_ptr) : herr_t;
+
+    extern proc H5Pget_mcdt_search_cb(plist_id : hid_t, ref func : H5O_mcdt_search_cb_t, ref op_data : c_void_ptr) : herr_t;
+
+    extern proc H5Pregister1(cls_id : hid_t, name : c_string, size : size_t, def_value : c_void_ptr, prp_create : H5P_prp_create_func_t, prp_set : H5P_prp_set_func_t, prp_get : H5P_prp_get_func_t, prp_del : H5P_prp_delete_func_t, prp_copy : H5P_prp_copy_func_t, prp_close : H5P_prp_close_func_t) : herr_t;
+
+    extern proc H5Pinsert1(plist_id : hid_t, name : c_string, size : size_t, value : c_void_ptr, prp_set : H5P_prp_set_func_t, prp_get : H5P_prp_get_func_t, prp_delete : H5P_prp_delete_func_t, prp_copy : H5P_prp_copy_func_t, prp_close : H5P_prp_close_func_t) : herr_t;
+
+    extern proc H5Pget_filter1(plist_id : hid_t, filter : c_uint, ref flags : c_uint, ref cd_nelmts : size_t, cd_values : c_ptr(c_uint), namelen : size_t, name : c_ptr(c_char)) : H5Z_filter_t;
+
+    extern proc H5Pget_filter_by_id1(plist_id : hid_t, id : H5Z_filter_t, ref flags : c_uint, ref cd_nelmts : size_t, cd_values : c_ptr(c_uint), namelen : si
```

Comment:
Any reason not to put this content in the outer `module HDF5`? It may be odd for users to have to find the `HDF5.HDF5_Chapel` page for documentation on the `HDF5` module.

Reply:
The `HDF5.HDF5_Chapel` module is a holdover from the previous organization of the `HDF5` module.  Removing it and moving its contents to the top level sounds good to me.